lighthouse-extension/gulpfile.js

```diff
@@ -100,6 +100,8 @@ gulp.task('chromeManifest', () => {
 function applyBrowserifyTransforms(bundle) {
   // Fix an issue with imported speedline code that doesn't brfs well.
   return bundle.transform('./fs-transform', {global: true})
+  // Replace url module with url-shim
+  .transform('./url-transform', {global: true})
```

Comment:
is it not possible to get the `bundle.require` style working like we do for audits/gatherers? I don't know if it's possible really just curious :)